src/pineko/evolve.py

```diff
@@ -145,13 +132,29 @@ def evolve_grid(
     # rotate to evolution (if doable and necessary)
     if np.allclose(operators.rotations.inputpids, br.flavor_basis_pids):
         eko.io.manipulate.to_evol(operators)
-    elif not np.allclose(operators.rotations.inputpids, br.evol_basis_pids):
+    elif not np.allclose(operators.rotations.inputpids, br.rotate_flavor_to_evolution):
         raise ValueError("The EKO is neither in flavor nor in evolution basis.")
     # do it
     order_mask = pineappl.grid.Order.create_mask(grid.orders(), max_as, max_al)
     muf2_grid = operators.mu2grid
     # PineAPPL wants alpha_s = 4*pi*a_s
     # remember that we already accounted for xif in the opcard generation
+    # legacy_class = eko.io.runcards.Legacy(operators.theory_card, {})
+    # new_tcard = legacy_class.new_theory
```

Comment:
Remember to remove void code before merging :upside_down_face:

Reply:
Yes, I still need to polish everything :)